testsuite/features/step_definitions/common_steps.rb

```diff
@@ -318,13 +318,14 @@
   arch, _code = node.run('uname -m')
   arch.chomp!
   dir_file = client_raw_repodata_dir("test-channel-#{arch}")
-  raise "File #{dir_file}/#{file} not exist" unless file_exists?(node, "#{dir_file}/#{file}")
+  _out, code = node.run("ls -1 #{dir_file}/*#{file} 2>/dev/null| wc -l")
```

Comment:
rather than a pipe on the node, it is more efficient to analyze the result of the `ls` in ruby. But ok.

Reply:
I changed it but I have no idea if it works. I will try to test my changes before I merge

Reply:
works